alacritty/src/config/ui_config.rs

```diff
@@ -41,8 +41,9 @@ pub struct UIConfig {
     /// Bindings for the mouse.
     mouse_bindings: MouseBindings,
 
-    /// Background opacity from 0.0 to 1.0.
-    background_opacity: Percentage,
+    /// TODO: Deprcated
```

Comment:
This annotation isn't necessary anymore, the new `#[config(deprecated)]` annotation is sufficient to fiend deprecated fields.

Reply:
Ah, neat, also removes my typo o_O..

Reply:
Dropped.